crates/sncast/src/helpers/configuration.rs

```diff
@@ -1,19 +1,19 @@
 use super::constants::{CONFIG_FILENAME, WAIT_RETRY_INTERVAL, WAIT_TIMEOUT};
+use crate::ValidWaitParams;
 use anyhow::{anyhow, Result};
 use camino::Utf8PathBuf;
 use serde::{Deserialize, Serialize};
 use std::fs;
 use tempfile::TempDir;
 use toml::Value;
 
-#[derive(Deserialize, Serialize, Clone, Debug, PartialEq)]
+#[derive(Default, Deserialize, Serialize, Clone, Debug, PartialEq)]
```

Comment:
just to be sure you've checked - a default value for an Option is `None`, right? (I'm thinking about keystore param)

Reply:
yeah

and `#[derive(Default)]` was suggested by clippy